src/Sylius/Bundle/UserBundle/Resources/config/doctrine/model/User.orm.xml

```diff
@@ -25,9 +25,9 @@
         <field name="encoderName" column="encoder_name" type="string" nullable="true" />
 
         <field name="lastLogin" column="last_login" type="datetime" nullable="true" />
-        <field name="passwordResetToken" column="password_reset_token" type="string" nullable="true" />
+        <field name="passwordResetToken" column="password_reset_token" type="string" unique="true" nullable="true" />
         <field name="passwordRequestedAt" column="password_requested_at" type="datetime" nullable="true" />
-        <field name="emailVerificationToken" column="email_verification_token" type="string" nullable="true" />
+        <field name="emailVerificationToken" column="email_verification_token" type="string"  nique="true" nullable="true" />
```

Comment:
```suggestion
        <field name="emailVerificationToken" column="email_verification_token" type="string"  unique="true" nullable="true" />
```

Reply:
It's very odd that the tests don't fail with that.

Reply:
> Fixed it manually. It's very odd that the tests don't fail with that.

Yep, no idea how it passed.